docs/_src/_includes/layouts/base.njk

```diff
@@ -7,6 +7,13 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge"/>
     <meta name="google-site-verification" content="TPIn3qa-ZOQsCQrpOB6J5eALE4dPBPiRBEJP5azpo9g"/>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, viewport-fit=cover"/>
+    <meta property="og:title" content="Bearer CLI: {{ title | escape }}"/>
+    <meta property="og:description" content="{{ meta.description | escape }}"/>
+    <meta property="og:image" content="/assets/img/bearer-cli-opengraph.png"/>
```

Comment:
Think you need to use the canonical url here ?

Reply:
`og:url` is on 13, but I've gone ahead and added the canonical rel as well 👍

Reply:
I meant as in the full url so "https://docs.bearer.sh/assets/img/bearer-cli-opengraph.png"

Reply:
Gotcha